packages/endpoints-organizations-registrar/src/entities/organizations/orchestrators/init-create-organization.js

```diff
@@ -76,13 +78,17 @@ const initCreateOrganization = (fastify) => {
       context
     );
 
-    const consents = buildServicesConsents(
-      organization,
-      organization.services,
-      context
-    );
-    if (!isEmpty(consents)) {
-      await repos.registrarConsents.insertMany(consents);
+    if (!isEmpty(organization.services)) {
+      const consents = map((service) => {
+        return {
+          userId: user.sub,
+          organization,
+          type: getServiceConsentType(service),
+          version: 1,
+          serviceId: service.id,
+        };
+      }, organization.services);
+      await repos.registrarConsents.registerConsent(consents);
```

Comment:
this should be a separate function `reigsterConsents`. I think we should avoid overloading

Reply:
done, [b51e9fe](https://github.com/velocitycareerlabs/velocitycore/pull/34/commits/b51e9fe7fad6cf0e21345db0d601f050ba20028e)